src/components/blocked-patron/isPatronBlockedHoc.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState, ComponentType, FC } from "react";
+import { useDispatch } from "react-redux";
+import { useGetPatronInformationByPatronIdV2 } from "../../core/fbs/fbs";
+import BlockedModal from "./blocked-modal/BlockedModal";
+import { AuthenticatedPatronV6 } from "../../core/fbs/model";
+import { useModalButtonHandler } from "../../core/utils/modal";
+import { getModalIds } from "../../core/utils/helpers/general";
+import { setHasBeenVisible } from "../../core/blockedModal.slice";
+import { RootState, useSelector } from "../../core/store";
+import BlockedTypes from "../../core/utils/types/BlockedTypes";
+import { redirectTo } from "../../core/utils/helpers/url";
+
+export interface PatronProps {
+  patron: AuthenticatedPatronV6 | null | undefined;
+}
+
+type InputProps = {
+  // This should be a redirect to the front page.
+  redirectOnBlocked: string;
+};
+
+// Hoc that determines if a patron is blocked and provides a modal with
+// and explanation for the user.
+const isPatronBlockedHoc =
```

Comment:
All the other HOC's we have created starts with: "with*". And are placed in: src/core/utils/.
I think we should keep that pattern.

Reply:
Renamed and moved